client/src/payment/deliveryInfor.js

```diff
@@ -0,0 +1,147 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
+import TextField from '@material-ui/core/TextField';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Select from '@material-ui/core/Select';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+
+
+
+const styles = theme => ({
+    container: {
+        display: 'flex',
+        flexWrap: 'wrap',
+    },
+    textField: {
+        marginLeft: theme.spacing.unit,
+        marginRight: theme.spacing.unit,
+        width: 200,
+    },
+    table: {
+        minWidth: 700,
+    },
+    menu: {
+        width: 200,
+    },
+    selectEmpty: {
+        marginTop: theme.spacing.unit * 2,
+    },
+    formControl: {
+        margin: theme.spacing.unit,
+        minWidth: 120,
+    },
+});
+
+
+class DeliveryInfor extends Component {
```

Comment:
![refactor](https://user-images.githubusercontent.com/5608395/44619664-d465bf00-a8cd-11e8-8f61-6bbe7c35ed47.png)

Should be DeliveryInfo

Reply:
done